docs/part3.md

```diff
@@ -1 +1,178 @@
 # Part III: Numerical precision in _ab initio_ calculations.
+
+In this part you will learn about several new quantities and tools for working with numerical convergence in NOMAD.
+These tools will help you
+
+1. query for the level of precision or conformity that you need.
+2. deploy extracted data into a notebook to start performing data science.
+
+Your starting point is the Entries page > FILTERS (side menu) > Precision.
+To navigate to the Entries page, check out [Part I - Exploring NOMAD](part1.md#entries_section).
+
+<!-- Screenshot of the Precision section -->
+
+Since these quantities are new and might not be commonly known, the emphasis is placed here on their definitions, as well as, the bare minimum of theoretical knowledge required to handle them.
```

Comment:
I think this paragraph is a bit vague. Say more directly "hey these quantities were agreed, you can find them here and here. Some boxes on the search filters help you."

Reply:
How do you mean "hey these quantities were agreed"?